airflow/providers/mongo/CHANGELOG.rst

```diff
@@ -27,6 +27,17 @@
 Changelog
 ---------
 
+3.7.0
+......
```

Comment:
Post merge comment:

I guess the version should be 4.0.0 in case of breaking changes

Reply:
Ah .. Right 🤦

Reply:
I will raise a follow up for this. Sorry missed it